plugins/uf2/README.md

```diff
@@ -0,0 +1,61 @@
+# UF2 Devices
+
+## Introduction
+
+This plugin allows the user to update any supported UF2 Device by writing
+firmware onto a mass storage device.
+
+A UF2 device exposes a VFAT block device which has a virtual file
+`INFO_UF2.TXT` where metadata can be read from. It may also have a the
+current firmware exported as a file `CURRENT.UF2` which is in a 512
+byte-block UF2 format.
+
+Writing any file to the MSD will cause the firmware to be written.
+Sometimes the device will restart and the volume will be unmounted and then
+mounted again. In some cases the volume may not “come back” until the user
+manually puts the device back in programming mode.
+
+Match the block devices using the VID*PID, UUID or label, and then create a
```

Comment:
uuid and label are gone now

Reply:
I kept UUID as a postfix to one of the instance IDs. I've removed label.